scripts/run-nodes.sh

```diff
@@ -96,11 +96,13 @@ function run-node () {
             --hostname=127.0.0.1 \
             --node-id=$NID \
             --test-miners=$N \
-            --chainweb-version=testWithTime \
+            --chainweb-version=testnet00 \
```

Comment:
Likely problematic, here and below.

Reply:
Oh, good catch! this was only for debugging, it wasn't meant to be submitted.